setup.py

```diff
@@ -32,7 +32,7 @@ def load_requirements(*requirements_paths):
     return list(requirements)
 
 setup(
-    name='ora2',
+    name='edx-ora2',
```

Comment:
This is what gets reported from `pip freeze`, am I correct in thinking we may want the `edx-` prefix there for easier grepping?